internal/charm/meta_test.go

```diff
@@ -83,14 +83,14 @@ func (s *MetaSuite) TestValidTermFormat(c *gc.C) {
 	for i, s := range valid {
 		c.Logf("valid test %d: %s", i, s)
 		meta := charm.Meta{Terms: []string{s}}
-		err := meta.Check(charm.FormatV1)
```

Comment:
Why did these change?

Reply:
Keep in mind this was written before our discussion at yesterday's standup

This change was made because I intended to drop support for FormatV1, so we error out if FormatV1 is provided.

To ensure these tests still function in some way, we pass in `charm.FormatV2` and `charm.SelectionManifest` as if the metadata we are Checking came from a charm with a valid manifest

This is subject to change though, depending on where we settle with FormatV1